Readme.md

```diff
@@ -15,6 +15,22 @@ and [Open Policy Agent WebAssembly dotnet core SDK](https://github.com/me-viper/
 
 We want **fast**, **in-process** and **secure** OPA policies evaluation, and avoid network bottlenecks when using [opa-java](https://github.com/StyraInc/opa-java).
 
+Using this integration you can switch from the traditional integration pattern:
```

Comment:
We could perhaps point out here that this currently only provides the evaluation part of OPA, not management features like decision logging, discovery bundles, etc. Just to set the expectations straight :)

Reply:
Make sense, let me know if the new wording is enough

Reply:
It's fine. I might come back later with a PR to expand a bit on this, if you don't mind 🙂

Reply:
That's appreciated @anderseknert :+1: thanks!